app/templates/components/job-controls.hbs

```diff
@@ -6,16 +6,20 @@
     {{#if error}}
       {{#bs-alert type="danger"}}<span class="error-message">{{error}}</span>{{/bs-alert}}
     {{/if}}
-    {{#if job.isNew}}
+    {{#if job.isAuthorized}}
```

Comment:
Right, but `job.isAuthorized` is only true when the job state is `A`. So if the job state is cancelled `C` or errored `E`, they will be prompted for a token instead of shown the job controls, right?